runtime/gc_base/GCExtensions.hpp

```diff
@@ -186,8 +186,6 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 	double maxRAMPercent; /**< Value of -XX:MaxRAMPercentage specified by the user */
 	double initialRAMPercent; /**< Value of -XX:InitialRAMPercentage specified by the user */
 
-protected:
-private:
```

Comment:
I would not remove these lines but rearrange code itself with extra comments. The reason why this lines are there it is an attempt to follow coding standards. For class definition it should be:
```
/* Class data members */
private:
protected:
public:
/* Class functions */
private:
protected:
public:
```
So comments are missed (no even empty line between sections) and private/protected/public are not in order and this creates a confusion.
However there is no functional change at all in this file so I would exclude it from PR

Reply:
I see, I'll remove these changes then, thanks!